libpyvinyl/Parameters/Collections.py

```diff
@@ -1,6 +1,8 @@
 # Created by Mads Bertelsen and modified by Juncheng E
 
-import json
+import json_tricks as json
+from collections import OrderedDict
```

Comment:
just another thought. Is this consistent with the uses of `to_dict()`? I guess we're only using `to_dict()` for preparation of a json dump but wanted to make sure.

Reply:
I will check.

Reply:
It seems to me ok.